subgraph.template.yaml

```diff
@@ -488,7 +488,7 @@ dataSources:
     name: SettingsInterface
     network: {{network}}
     source:
-      startBlock: {{startBlock_Settings}}
+      startBlock: {{startBlock}}
```

Comment:
This startBlock would be per network right @salazarguille ,
I think for the compound markets stuff this won't be useful, since we require the min blockNumber for mainnet :disappointed:

Reply:
The settings contract was growing and we needed to "refactor" it to decrease the size. Now, we are using a mapping key=> {value, min, max} (instead of property -number- for each setting). The key is a string value (the setting name), and when we want to update the value, the new value is validated (the min and max). Before this change, we did all the validations per setting (it means 2 functions per setting -like copy/paste and changing the property name-). Now we use this generic way. It means, if we need to create a new setting, we just call the createPlatformSetting function (with the name, value, min, and max).

Reply:
The startBlock is to tell TheGraph what is the starting point (block number). It is per network (/config/{network}.js file)

I think you are referring to "startBlockNumber" and "startBlockOffsetNumber" settings that you will use in the event listener. If so, they are different "settings". One is for TheGraph (to make the syncing process faster), and another one is the platform setting/s.